sap-successfactors/4.1/modules/ROOT/pages/sap-successfactors-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,77 @@
+= Upgrade SAP SuccessFactors Connector to Version 4.0 - Mule 4
```

Comment:
I want to discuss how to deal with update topics like this one at a connector meeting. We should probably change 4.0 to 4.x and add the new functionality for the minor release but don't do anything now.